packages/design-system/src/styles/components/_Table.scss

```diff
@@ -71,7 +86,7 @@
     border: 0;
     border-top: 1px solid;
     display: grid;
-    grid-template-columns: 120px 1fr;
+    grid-template-columns: minmax(0, $table-stacked-title-max-width) 1fr;
```

Comment:
Oh whats this `minmax` function? I've never seen it before

Reply:
Using minmax to fix the inconsistent word-wrap issue for grid 
https://developer.mozilla.org/en-US/docs/Web/CSS/minmax 
https://css-tricks.com/css-grid-in-ie-debunking-common-ie-grid-misconceptions/

Reply:
oh cool! nice find for the fix. Could you add a comment briefly summarizing the issue and why you need it? Including the css-tricks link would be helpful too

Reply:
Is this the most relevant part?
https://css-tricks.com/css-grid-in-ie-debunking-common-ie-grid-misconceptions/#fit-content-is-not-ie-friendly-but
we are able to get `fit-content` like behavior with minmax?